nengo/builder/tests/test_transforms_conv.py

```diff
@@ -88,6 +88,93 @@ def test_convinc_2d(
     assert allclose(signals[y], y0)
 
 
+@pytest.mark.parametrize("channels_last", (True, False))
+@pytest.mark.parametrize("padding", ("same", "valid"))
+@pytest.mark.parametrize("groups", (1, 8, 32))
+@pytest.mark.parametrize("out_channels", (32, 64))
+def test_grouped_convinc_2d(
+    channels_last, padding, groups, out_channels, rng, allclose
+):
+    correlate2d = pytest.importorskip("scipy.signal").correlate2d
+
+    shape0, shape1 = 16, 17
```

Comment:
What I was thinking for the test is that we could have one `_test_convinc_2d` helper function, that is called by both `test_convinc_2d` and `test_grouped_convinc_2d`. That way, there's not all the duplicated code.